### PR TITLE
Use legacy-report in 2692 changes

### PR DESCRIPTION
**Pull request by billli0**
### Issue #:
[#2913](https://github.com/beyondessential/tupaia-backlog/issues/2913)
Previous PR: https://github.com/beyondessential/tupaia/pull/2692

This 2913 ticket is already merged, but needs to update to use new db dashboard structure.

### CODE REVIEW

File: packages/database/src/migrations/20210621034818-FixSbAtLeastOneStaffMemberTrained-modifies-data.js [hunk 1]
```diff
@@ -14,27 +14,27 @@ exports.setup = function (options, seedLink) {
...
 exports.up = async function (db) {
-  for (const dashboardId of dashboardIds) {
+  for (const dashboardItemId of dashboardItems) {
     await db.runSql(`
-      update "dashboardReport" dr
-      set "dataBuilderConfig" = regexp_replace(dr."dataBuilderConfig"::text, 'RHS1UNFPA03','RHS4UNFPA807','g')::jsonb
-      where id = '${dashboardId}'
+      update "legacy_report" dr
+      set "data_builder_config" = regexp_replace(dr."data_builder_config"::text, 'RHS1UNFPA03','RHS4UNFPA807','g')::jsonb
+      where id = '${dashboardItemId}'
```
Comment (chris-bes) [inline packages/database/src/migrations/20210621034818-FixSbAtLeastOneStaffMemberTrained-modifies-data.js:27]: This is code now
```suggestion
      where code = '${dashboardItemId}'
```

File: packages/database/src/migrations/20210621034818-FixSbAtLeastOneStaffMemberTrained-modifies-data.js [hunk 2]
```diff
@@ -14,27 +14,27 @@ exports.setup = function (options, seedLink) {
...
 exports.down = async function (db) {
-  for (const dashboardId of dashboardIds) {
+  for (const dashboardItemId of dashboardItems) {
     await db.runSql(`
-      update "dashboardReport" dr
-      set "dataBuilderConfig" = regexp_replace(dr."dataBuilderConfig"::text, 'RHS4UNFPA807','RHS1UNFPA03','g')::jsonb
-      where id = '${dashboardId}'
+      update "legacy_report" dr
+      set "data_builder_config" = regexp_replace(dr."data_builder_config"::text, 'RHS4UNFPA807','RHS1UNFPA03','g')::jsonb
+      where id = '${dashboardItemId}'
```
Comment (chris-bes) [inline packages/database/src/migrations/20210621034818-FixSbAtLeastOneStaffMemberTrained-modifies-data.js:37]: Same as above

**Review by chris-bes — CHANGES_REQUESTED** [review]

File: packages/database/src/migrations/20210621034818-FixSbAtLeastOneStaffMemberTrained-modifies-data.js (hunk 1 above)
Comment (billli0) [reply packages/database/src/migrations/20210621034818-FixSbAtLeastOneStaffMemberTrained-modifies-data.js:27]: right, thanks @chris-bes !

**Review by chris-bes — APPROVED** [review]
Also tested locally, works fine!

**Pull request merged**